src/style.css

```diff
@@ -0,0 +1,102 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+body {
+  margin-top: 0;
+}
+
+.formBig {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.searchBig {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  padding-left: 62px;
+  line-height: 22px;
+  font-size: 16px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.photo {
+  position: absolute;
+  width: 19px;
+  bottom: 26px;
+  left: 26px;
+}
+
+.searchBig:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchBig:focus {
+  outline: none;
+  font-family: "AvenirBold", sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.searchBig::placeholder {
+  font-family: "Avenir", sans-serif;
+}
+
+.formSmall {
+
+  position: relative;
+}
+
+.searchSmall {
+  height: 42px;
+  box-sizing: border-box;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  padding-left: 33px;
+  line-height: 19px;
+  font-size: 14px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.photoSmall {
+  width: 11px;
+  bottom: 16px;
+  left: 13px;
+}
+
+.searchSmall:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchSmall:focus {
+  outline: none;
+  font-family: "AvenirBold", sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.searchSmall::placeholder {
+  font-family: "Avenir", sans-serif;
+}
```

Comment:
[Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 2nd item.

Reply:
Good advice !  Non specific properties has been redistributed to parents.